### ISSUE TITLE
[Suggestion] Add support to download replays across servers.

### ISSUE THREAD

**Issue by krudge137** [dejan9393/lol-run-it-back#44]
Currently I'm unable to download replays from other servers that I'm not logged into. If it's possible to make it work globaly it would be more than perfect.

**Comment by arthures11** [comment]
looking for similiar solution

**Comment by krudge137** [comment]
> looking for similiar solution

we probably aint getting it

**Comment by arthures11** [comment]
yeah i mean - is it even possible? to download/run a replay from the server youre not logged in? dunno

**Comment by krudge137** [comment]
I think its possible, but what do I know 🤣, that's why I created this issue hopefuly someday it will be implemented ( if possible ofc )
